src/web_interface/templates/show_analysis.html

```diff
@@ -178,15 +184,6 @@ <h3>
                                     </label>
                                 </div>
                             </div>
-                            <script>
-                                function start_compare(){
-                                    var checkbox = document.getElementById('recompare_checkbox');
-                                    var link = '/compare';
-                                    if (checkbox.checked) link += '?force_recompare=true';
-                                    location.href = link;
-                                };
-                                document.getElementById("start_compare_button").onclick = start_compare;
```

Comment:
It seems this line got lost somewhere on the way (and therefore the compare button does not work anymore)

Reply:
Oh... But the line alone doesn't matter. The start_compare.js needs to be placed where the code was taken